pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -315,6 +319,9 @@ func ConfigPath(ctx *types.SystemContext) string {
 			confPath = filepath.Join(ctx.RootForImplicitAbsolutePaths, systemRegistriesConfPath)
 		}
 	}
+	if _, err := os.Stat(filepath.Join(homedir.Get(), userRegistriesFile)); err == nil {
```

Comment:
- `SystemContext`, if it set, has precedence over everything
- Then per-user, if exists (and taking into account `RootForImplicitAbsolutePaths`)
- Then the system-wide.

This seems to prefer the per-user file to everything else.

(Non-blocking, but updating `TestConfigPath` would be nice, and would have demonstrated this.)